youtube_dl/extractor/__init__.py

```diff
@@ -18,11 +18,11 @@
 from .aftonbladet import AftonbladetIE
 from .airmozilla import AirMozillaIE
 from .aljazeera import AlJazeeraIE
+from .allocine import AllocineIE
```

Comment:
These changes (alphabetic reordering) are not related to qq extractor thus should not be present in this PR.